pkg/proc/target.go

```diff
@@ -36,8 +36,12 @@ const (
 // Target represents the process being debugged.
 type Target struct {
 	Process
+	RecordingManipulation
 
-	proc ProcessInternal
+	proc   ProcessInternal
+	recman RecordingManipulationInternal
+
+	Pid int
```

Comment:
I'd actually prefer this remain unexported and we expose it via a method. It makes it clear that it should never be modified (and makes it impossible to modify) via any consumer package.

Reply:
Done.